app/utility-analysis/page.tsx

```diff
@@ -4,14 +4,14 @@ import { createClient } from '@/utils/supabase/client';
 
 const UtilityAnalysis = async () => {
   // Supabase Client erstellen
-  const supabase = createClient();
+  const supabase = await createClient();
```

Comment:
Here, the `await` was missing. As you can see in the docs of supabase, the `createClient` is an asynchronous call and needs to be awaited.